frontend/src/store/pinia-modules/oauth.ts

```diff
@@ -13,10 +13,37 @@ const convert = (raw: ResourceObject): OAuthToken => {
 
 export const useOAuthStore = defineStore("oauth", {
   actions: {
-    async exchangeVCSToken({ vcsId, code }: { vcsId: VCSId; code: string }) {
+    // Either pass the "vcsId" to let the backend infer the details from an existing VCS provider
+    // or pass "vcsType", "instanceURL", "clientId" and "clientSecret" to allow the backend directly
```

Comment:
Maybe split to two methods, exchangeVCSTokenWithID and exchangeVCSTokenWith(ClientInfo?) ?

Also comment on exchangeVCSTokenWithClientInfo that it's only used for initial VCS setup.

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/1007/commits/bfd578c27922d3c1a46bc878b28c58bcd2d621c0.